src/services/doppler-contact-policy-api-client.test.ts

```diff
@@ -0,0 +1,15 @@
+import { cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('HttpDopplerContactPolicyApiClient', () => {
+  afterEach(cleanup);
+
+  //TODO: Implement real tests
```

Comment:
Will it be done before or after the merge? 😛

Reply:
After

Reply:
😕 Mmmm, in that case, I think that you could remove the file, it is totally unuseful at the moment.

Or..., you can add tests before the merge, you can mock Axios response and test the URLs requested, the mappers, how you deal with the errors, etc.